docs/resources/Guild.md

```diff
@@ -148,6 +159,7 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
 | nick? | string | this users guild nickname (if one is set) |
 | roles | array of snowflakes | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids |
 | joined_at | ISO8601 timestamp | when the user joined the guild |
+| premium_since | ?ISO8601 timestamp | when the user subscribed to the guilds premium |
```

Comment:
nit:

`when the user subscribed to the guilds premium` --> `when the user used their Nitro boost on the server`

Since "subscribing to guild's premium" isn't really what the feature is

Reply:
This nit also makes more sense grammatically and from a perspective lacking context.